reaper/reaper_test.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 The go-zeromq Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package reaper
```

Comment:
wouldn't this work as well ? (and put it under the root of zmq4)
```suggestion
package zmq4_test
```

Reply:
No it wont works - the zmq4_test has init() which allocates sockets/goroutines, and goleak complains about those. So this test has to be fully excluded. atm I did it this way

Reply:
Renamed to leaks_test
May be that would be better